helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -56,18 +58,15 @@ internal class SplitCommand : CliktCommand(
         .required()
 
     override fun run() {
-        val packageCurations = inputCurationsFile.readValue<List<PackageCuration>>()
+        val packageCurations = readPackageCurations(inputCurationsFile)
+
         val groupedCurations = packageCurations.groupBy {
             getSplitCurationFile(outputCurationsDir, it.id, inputCurationsFile.extension)
         }
 
         val mapper = createBlockYamlMapper()
         groupedCurations.forEach { (outputFile, curations) ->
-            val curationsToPersist = if (outputFile.isFile) {
-                outputFile.readValue<MutableSet<PackageCuration>>()
-            } else {
-                mutableSetOf()
-            }
+            val curationsToPersist = readPackageCurations(outputFile).toMutableSet()
```

Comment:
Please add a rationale to the commit message, esp. as there seems no reuse being involved, and it's only a small amount of code anyway. The "reuse" in line 61 is not required as the input file is forced to exist.